packages/uppy/index.mjs

```diff
@@ -20,6 +20,7 @@ export { default as StatusBar } from '@uppy/status-bar'
 
 // Acquirers
 export { default as Dropbox } from '@uppy/dropbox'
+export { default as Box } from '@uppy/box'
```

Comment:
excluding box from global module for now as it will initially be released as beta
```suggestion
```